src/Components/Map/Map.js

```diff
@@ -138,9 +133,6 @@ const cleanYelpResponse = yelp => {
             updateStops={updateStops}
             waypoints={waypoints}
             stops={stops}
-            map={currentMap}
```

Comment:
Thank you for taking these out I forgot we had them in there, make the pin components look much cleaner